apiserver/Makefile

```diff
@@ -73,7 +73,7 @@ endif
 .PHONY: build
 build: $(BINDIR)/apiserver-$(ARCH)
 
-bin/LICENSE: ../LICENSE.md
+$(BINDIR)/LICENSE: ../LICENSE.md
```

Comment:
I'd prefer if we used `bin/` here like it was before and have the Dockerfile look in `bin` - BINDIR varies based on architecture, and we don't need a license per architecture.